EIPS/eip-1.md

```diff
@@ -232,6 +232,64 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Other Well Known Standards Bodies
+
+Links to other well known standards bodies specification may be included using normal markdown syntax. Be mindful of the requirements set for each SDO as these were chosen in order to maintain links to stable URLs.
+
+#### World Wide Web Consortium (W3C)
+
+Links to a W3C "Recommendation" status specification may be included using normal markdown syntax, such as:
+
+```markdown
+[W3C Secure Context](https://www.w3.org/TR/secure-contexts/)
+```
+
+Which renders as:
+
+[W3C Secure Context](https://www.w3.org/TR/secure-contexts/)
+
+Permitted W3C recommendation URLs MUST anchor to a specification in the technical reports namespace, and so MUST match this regular expression:
+
+```regex
+^https:\/\/www.w3.org\/TR\/.*$
+```
+
+#### Internet Engineering Task Force (IETF)
+
+Links to an Request For Comment (RFC) specification may be included using normal markdown syntax, such as:
+
+```markdown
+[RFC-8446](https://www.rfc-editor.org/rfc/rfc8446)
+```
+
+Which renders as:
+
+[RFC-8446](https://www.rfc-editor.org/rfc/rfc8446)
+
+Permitted IETF specification URLs MUST anchor to a specification with an assigned RFC number (meaning cannot reference internet drafts), and so MUST match this regular expression:
+
+```regex
+^https:\/\/www.rfc-editor.org\/rfc\/.*$
+```
+
+#### Web Hypertext Application Technology Working Group (WHATWG)
+
+Links to WHATWG specification may be included using normal markdown syntax, such as:
+
+```markdown
+[WHATWG HTML Spec](https://html.spec.whatwg.org/)
+```
+
+Which renders as:
+
+[WHATWG HTML Spec](https://html.spec.whatwg.org/)
+
+Permitted WHATWG specification URLs MUST anchor to a specification defined in the spec subdomain (idea specifications are not allowed), and so MUST match this regular expression:
+
+```regex
+^https:\/\/[a-z]*.spec.whatwg.org.*$
+```
```

Comment:
```suggestion
Links to versions of WHATWG specifications may be included using normal markdown syntax, where the content of the link is the title of the specification, followed by ` Specification` if applicable. For example, the following link would be allowed:

```markdown
[HTML Specification](https://html.spec.whatwg.org/commit-snapshots/3c1c2541831c53cd7c47c0c8883cd5458d80e750/)
a```

Which renders as:

[HTML Specification](https://html.spec.whatwg.org/commit-snapshots/3c1c2541831c53cd7c47c0c8883cd5458d80e750/)

Permitted WHATWG specification URLs MUST anchor to a specification defined in the spec subdomain (idea specifications are not allowed), and must be snapshots. They MUST match this regular expression:

```regex
^https:\/\/[a-z]*.spec.whatwg.org\/commit-snapshots\/[0-9a-f]{40}$
a```
```

Reply:
This one I'm a bit hesitant about, I'm a +0 because to linking to commit-snapshots because the advantage to WHATWG docs is that they're living standards. In general this means that breaking changes aren't happening and there's a desire to maintain backwards compatibility so that the web isn't being broken. This is why when you show up to a commit-snapshot link they show:

<img width="1492" alt="Screenshot 2023-03-17 at 2 22 37 PM" src="https://user-images.githubusercontent.com/23125059/226000148-744789c3-8cb1-423f-8791-561459994bc1.png">

Reply:
Let me see if I can dig up any rules on this to see what tradeoffs appear with WHATWG standards. From what I understand they have some different governance process in order to update a spec which are lighter weight and don't involve membership requirements so this seems like it would be useful context for us to understand anyways since it seems well aligned with the direction EIPs want to be going.

Reply:
Fair enough. I'll allow linking to full living specs for now, but I would like EIP-1 to eventually be updated with rules of how linking to living specs should be handled.

I could honestly see a world in which Living EIPs become more common, even for non-meta non-informational standards. I wouldn't be entirely opposed to that. Anyway, I'm getting off-topic :)

```suggestion
Links to WHATWG specifications may be included using normal markdown syntax, where the content of the link is the title of the specification, followed by ` Specification` if applicable. For example, the following link would be allowed:

```markdown
[WHATWG HTML Spec](https://html.spec.whatwg.org/)
```

Which renders as:

[WHATWG HTML Spec](https://html.spec.whatwg.org/)

Permitted WHATWG specification URLs MUST anchor to a specification defined in the spec subdomain (idea specifications are not allowed), and so MUST match this regular expression:

```regex
^https:\/\/[a-z]*.spec.whatwg.org.*$
```
```